virt/virt-getting-started.adoc

```diff
@@ -0,0 +1,80 @@
+* xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Prepare your cluster for {VirtProductName}].
+* xref:../virt/install/installing-virt-web.adoc#virt-installing-virt-operator_installing-virt-web[Install the {VirtProductName} Operator].
+* xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[Install the `virtctl` command line interface (CLI) tool].
+
+[discrete]
+[id="additional-resources-planning-and-installing"]
+[role="_additional-resources"]
+=== Planning and installation resources
+* xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider].
+* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage for virtual machines].
+* xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#installing-the-kubernetes-nmstate-operator-cli[Installing the Kubernetes NMState Operator].
+* xref:../virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc#virt-specifying-nodes-for-vms[Specifying nodes for virtual machines].
+* xref:../virt/virt-using-the-cli-tools.adoc#virt-virtctl-commands_virt-using-the-cli-tools[`Virtctl` commands].
+
+[id='configuring-networks_{context}']
+=== Configuring secondary networks
+
+Configuring a secondary network is optional. VMs are connected to the default pod network.
+
+Configure a Linux bridge network:
+
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-linux-bridge-nncp_virt-attaching-vm-multiple-networks[Create a node network configuration policy].
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-linux-bridge-nad-web_virt-attaching-vm-multiple-networks[Create a network attachment definition for the Linux bridge network].
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-vm-creating-nic-web_virt-attaching-vm-multiple-networks[Attach a VM to the Linux bridge network].
+
+Configure an SR-IOV network:
+
+* xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[Install the SR-IOV Operator].
+* xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[Configure an SR-IOV network device].
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[Attach a VM to the SR-IOV network].
+
+[id="creating-and-managing-vms_{context}"]
+== Creating and managing virtual machines
+
+Create virtual machines (VMs) by using the web console:
+
+* xref:../virt/virtual_machines/virt-create-vms.adoc#virt-quick-creating-vm_virt-create-vms[Quick create a VM].
```

Comment:
The landing page that this link 'Quick create a VM' navigates to is the same as the one for 'Customize a template to create a VM'. If you have a link then it should navigate to the specific text. Seems redundant for it to navigate to another page with another link.

Reply:
The link and anchor ID work. However, the header, "Quick create a VM" is at the top of the screen and not clearly visible.